extern/sector-storage/ffiwrapper/sealer_cgo.go

```diff
@@ -546,7 +546,7 @@ func (sb *Sealer) FinalizeSector(ctx context.Context, sector abi.SectorID, keepU
 		defer done()
 
 		pf, err := openPartialFile(maxPieceSize, paths.Unsealed)
-		if xerrors.Is(err, os.ErrNotExist) {
+		if err != nil {
```

Comment:
I am not sure if `os.ErrNotExist` requires any special treatment here but ignoring other errors is clearly a bug.